symfony/framework-bundle/3.3/public/index.php

```diff
@@ -32,7 +22,7 @@
     Request::setTrustedHosts(explode(',', $trustedHosts));
 }
 
-$kernel = new Kernel($env, $debug);
+$kernel = new Kernel($_SERVER['APP_ENV'], $_SERVER['APP_DEBUG']);
```

Comment:
considering env variables can only be string, it feels wrong to use them in boolean context here. (even if it is casted to boolean in bootstrapEnv().

Reply:
I'm afraid is might teach bad practices when people just copy-paste such things for other variables without thinking about it. That is why the env processors got introduced. But not a big deal to me.

Reply:
works for me: APP_DEBUG is now normalized to the '1' or '0' string